app/src/main/java/org/mozilla/fenix/compose/ext/TabSessionState.kt

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose.ext
+
+import androidx.compose.runtime.Composable
+import mozilla.components.browser.state.state.TabSessionState
+import mozilla.components.browser.toolbar.MAX_URI_LENGTH
+import org.mozilla.fenix.components.components
+import org.mozilla.fenix.compose.inComposePreview
+import org.mozilla.fenix.ext.toShortUrl
+
+/**
+ * Indicates whether the app is currently running in a Composable @Preview.
+ */
+@Composable
+fun TabSessionState.toShortUrl(): String =
```

Comment:
an extension file already exists for `TabSessionState`, but wasn't sure since this function is only relevant to Compose, if it should live in its own `compose/ext` directory file